nibabel/freesurfer/tests/test_io.py

```diff
@@ -55,21 +56,31 @@ def test_geometry():
     assert_equal(coords.shape[0], faces.max() + 1)
 
     # Test quad with sphere
```

Comment:
Had a look at that file. It does have metadata at the end, but there is a big chunk (885354 bytes) before it. I'd say it's worth testing the quad file parsing, but not worth grabbing metadata from, since you had to go back to FSv3 to find it. So that can be handled in a different PR.

I think here I'd say just remove the misleading comment.